python_modules/libraries/dagster-fivetran/dagster_fivetran/translator.py

```diff
@@ -67,7 +67,44 @@ def to_fivetran_connector_table_props_data(self) -> Sequence[FivetranConnectorTa
         """Method that converts a `FivetranWorkspaceData` object
         to a collection of `FivetranConnectorTableProps` objects.
         """
-        raise NotImplementedError()
+        data: List[FivetranConnectorTableProps] = []
+
+        for connector_id, connector_data in self.connectors_by_id.items():
+            connector_details = connector_data.properties
+            connector_name = connector_details["schema"]
+            connector_url = get_fivetran_connector_url(connector_details)
+
+            schema_config = connector_details["schema_config"]
+
+            destination_details = self.destinations_by_id[
+                connector_details["destination_id"]
+            ].properties
+            database = destination_details.get("config", {}).get("database")
+            service = destination_details.get("service")
+
+            schemas_data = cast(Dict[str, Any], schema_config["schemas"])
+            for schema in schemas_data.values():
+                if schema["enabled"]:
+                    schema_name = schema["name_in_destination"]
+                    schema_tables: Dict[str, Dict[str, Any]] = cast(
```

Comment:
strong code smell from this one. The complexity from the previous PR feels like it's spilling over into this one with all the casting and raw string munging we need to do. Let's strongly type all of these objects, I think things will feel much better.

Reply:
Agreed. This [comment](https://github.com/dagster-io/dagster/pull/25788#discussion_r1834824869) applies here as well - this is taken from the legacy code to replicate the behavior. It will be updated in the same PR refactoring how we are storing destinations and connectors.